examples/scripts/dpo/dpo_tldr.py

```diff
@@ -0,0 +1,262 @@
+# flake8: noqa
```

Comment:
This file was to train a DPO model to reference SRPO against. I figured it couldn't hurt to have a TL;DR DPO script example anyway,

Reply:
we try to keep one script per trainer, can you please remove any example but `example/scripts/srpo.py`?
Ideally as close as possible to `example/scripts/dpo.py`